planetary-social/nos
# Show profile when search matches a valid npub

Shows the profile when the search is a valid npub. There's more work to do on this overall, but this meets the first requirement in #776.

I'd like to have this reviewed independently of some followup work I want to do on this. @mplorentz and I discussed moving some code from DiscoverView to SearchController as well as renaming a property or two, and I plan to do all that in my next PR.

I realize this is currently missing a CHANGELOG entry so I'll get that done soon.

Testing steps (on Mac):

1. Copy a valid profile npub id
2. Click the Discover tab
3. Click the search bar
4. Paste the npub id
5. Observe the profile
